iocage/lib/errors.py

```diff
@@ -391,13 +391,16 @@ def __init__(
         value: str,
         property_name: str,
         jail: typing.Optional[iocage.lib.Jail.JailGenerator]=None,
-        logger: typing.Optional[Logger]=None
+        logger: typing.Optional[Logger]=None,
+        level: str="error"
     ) -> None:
         reason = f"expected \"<nic>|<address>\" but got \"{value}\""
         super().__init__(
             property_name=property_name,
             jail=jail,
-            reason=reason
+            reason=reason,
+            level=level,
+            logger=logger
```

Comment:
there really should be an easier way to do this

Reply:
We just got rid of **kwargs for the sake of strong typing.